mappings/net/minecraft/world/EntityView.mapping

```diff
@@ -76,19 +76,39 @@ CLASS net/minecraft/class_1924 net/minecraft/world/EntityView
 		ARG 1 entityClass
 		ARG 2 box
 		ARG 3 predicate
-	METHOD method_8333 getEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+	METHOD method_8333 getSurroundingEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
```

Comment:
"surrounding" is really a bad name for this

Reply:
I'd say `getEntitiesWithin` or some amalgamation of that since the box specifies the volume to search within.